### DISCUSSION TITLE
Apple / Mac graphics cards

### DISCUSSION THREAD

**Discussion by Saul-Simhon** [mitsuba-renderer/mitsuba3#1016] [Q&A]
Is there a rendering variant optimized for Apple graphics cards, including recent MacBook pros which have hardware accelerated ray tracing?

**Comment by njroussel** [comment]
Hi, 

No, not yet. The status hasn't changed since[ this discussion](https://github.com/mitsuba-renderer/mitsuba3/discussions/180).
